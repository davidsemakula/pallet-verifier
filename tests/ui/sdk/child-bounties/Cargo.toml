[package]
name = "pallet-child-bounties"
version = "27.0.0"
#authors.workspace = true
#edition.workspace = true
edition = "2021"
license = "Apache-2.0"
#homepage.workspace = true
#repository.workspace = true
description = "FRAME pallet to manage child bounties"
readme = "README.md"

#[lints]
#workspace = true

[package.metadata.docs.rs]
targets = ["x86_64-unknown-linux-gnu"]

[dependencies]
codec = { version = "3.6.12", package = "parity-scale-codec", default-features = false, features = ["derive"] }
log = { version = "0.4.22", default-features = false }
scale-info = { version = "2.11.3", default-features = false, features = ["derive"] }
frame-benchmarking = { version = "38.0.0", optional = true, default-features = false }
frame-support = { version = "38.0.0", default-features = false }
frame-system = { version = "38.0.0", default-features = false }
pallet-bounties = { version = "37.0.0", default-features = false }
pallet-treasury = { version = "37.0.0", default-features = false }
sp-core = { version = "34.0.0", default-features = false }
sp-io = { version = "38.0.0", default-features = false }
sp-runtime = { version = "39.0.0", default-features = false }

[dev-dependencies]
pallet-balances = { version = "39.0.0", default-features = true }

[features]
default = ["std"]
std = [
	"codec/std",
	"frame-benchmarking?/std",
	"frame-support/std",
	"frame-system/std",
	"log/std",
	"pallet-balances/std",
	"pallet-bounties/std",
	"pallet-treasury/std",
	"scale-info/std",
	"sp-core/std",
	"sp-io/std",
	"sp-runtime/std",
]
runtime-benchmarks = [
	"frame-benchmarking",
	"frame-benchmarking/runtime-benchmarks",
	"frame-support/runtime-benchmarks",
	"frame-system/runtime-benchmarks",
	"pallet-balances/runtime-benchmarks",
	"pallet-bounties/runtime-benchmarks",
	"pallet-treasury/runtime-benchmarks",
	"sp-runtime/runtime-benchmarks",
]
try-runtime = [
	"frame-support/try-runtime",
	"frame-system/try-runtime",
	"pallet-balances/try-runtime",
	"pallet-bounties/try-runtime",
	"pallet-treasury/try-runtime",
	"sp-runtime/try-runtime",
]
