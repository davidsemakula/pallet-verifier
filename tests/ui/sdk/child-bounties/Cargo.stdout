Searching for FRAME pallet definition ...
Searching for dispatchable function definitions ...
Searching for implementations of pallet hooks ...
Searching for public associated function definitions ...
Searching for pallet associated function calls ...
Generating tractable entry points for FRAME pallet ...
Generating specialized summaries ...
Analyzing FRAME pallet with MIRAI ...
Creating summaries for FRAME and Substrate functions ...
Analyzing dispatchables ...
Analyzing dispatchable: `accept_curator` ...
Analyzing dispatchable: `add_child_bounty` ...
Analyzing dispatchable: `award_child_bounty` ...
Analyzing dispatchable: `claim_child_bounty` ...
Analyzing dispatchable: `close_child_bounty` ...
Analyzing dispatchable: `propose_curator` ...
Analyzing dispatchable: `unassign_curator` ...
Analyzing pub assoc fns ...
Analyzing pub assoc fn: `child_bounty_account_id` ...
Analyzing pub assoc fn: `<Self as pallet_bounties::ChildBountyManager>::child_bounties_count` ...
Analyzing pub assoc fn: `<Self as pallet_bounties::ChildBountyManager>::children_curator_fees` ...
